packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -157,6 +160,7 @@ export default function MainFeedLayout({
     setHidden,
   } = useDynamicLoadedAnimation();
   const [createMyFeed, setCreateMyFeed] = useState(false);
+  const [myFeedMode, setMyFeedMode] = useState('manual');
```

Comment:
What about enum like `FeedMode`, so we don't rely on hard concrete string values